### ISSUE TITLE
TOPICS: Participant view has a broken layout with misalignment and the side button missing (mobile)

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#1645]
**What is the problem?**
_Tested in iPhone 14 Plus in Safari_

In iPhone 14 Plus, clicking on Manage participants displays a view with misalignment (screenshot1), and a missing side button. The Remove button only displays if the user clicks the screen in a random place (screenshot2).

![screenshot1](https://github.com/user-attachments/assets/ef67b0eb-061d-4416-90cc-87b99140b13a)

![screenshot3](https://github.com/user-attachments/assets/92358694-0964-4378-9bc4-910011c74893)

**Why is this a problem?**
User will not be able to find the button on Mobile. Related to #1644

**Possible solution.**
Layout should be fixed and side button should display next to the participant

**Comment by ssin1901** [comment]
@BeccaMelhuish marked as "soon" but I think it needs to go in "next"?

**Comment by BeccaMelhuish** [comment]
@ssin1901 moved to next, should we add high priority label too?

**Comment by ssin1901** [comment]
@BeccaMelhuish absolutely, agreed it should be Next and high priority. Could you by any chance check whether it is happening on Android as well?

**Comment by BeccaMelhuish** [comment]
@ssin1901 for me on Android it's OK, clicking the 3 dots brings up the 'Remove member' :) 

![image](https://github.com/user-attachments/assets/e1fc1c95-bd9e-4f86-a161-f7f955121644)

**Comment by ssin1901** [comment]
@BeccaMelhuish thanks a million. I think we can remove the high priority label then, as it seems an iOS specific problem then.
